Infomaniak Public Cloud 01 (S3).cyberduckprofile

```diff
@@ -37,16 +37,8 @@
         <false/>
         <key>Anonymous Configurable</key>
         <false/>
-        <key>Region</key>
-        <string>auto</string>

```

Comment:
While we default to `us-east-1` it doesn't harm to explicitly specify it here.

Reply:
Add in all profiles except in "Infomaniak Public Cloud 01 (S3)" when this option is set in this profile. Bucket listing doesn't work.